### ISSUE TITLE
Due to error in Nebulosa package and ggplot2 v3.5.0

### ISSUE THREAD

**Issue by XIANZHE-LI** [samuel-marsh/scCustomize#190]
Plot_Density_Custom(seurat_object = DCM, 
+                     features = "SIRT1")
Error in `Plot_Density_Custom()`:
! Due to error in Nebulosa package and ggplot2 v3.5.0
  Plot_Density_Custom functionality is currently restricted to
  ggplot v3.4.4 or lower.
Run `rlang::last_trace()` to see where the error occurred.
> library(scCustomize)
> library(Nebulosa)
> Plot_Density_Joint_Only(seurat_object =DCM, 
+                         features =gene,
+                         reduction = "tsne"
+                         )
Error in `Plot_Density_Joint_Only()`:
! Due to error in Nebulosa package and ggplot2 v3.5.0
  Plot_Density_Joint_Only functionality is currently restricted to
  ggplot v3.4.4 or lower.
Run `rlang::last_trace()` to see where the error occurred.

####################################################
> library(scCustomize)
Loading required package: Seurat
Loading required package: SeuratObject
Loading required package: sp

Attaching package: ‘SeuratObject’

The following objects are masked from ‘package:base’:

    intersect, t

scCustomize v2.1.2
If you find the scCustomize useful please cite.
See 'samuel-marsh.github.io/scCustomize/articles/FAQ.html' for
citation info.
> library(Nebulosa)
Loading required package: ggplot2
Loading required package: patchwork
![image](https://github.com/user-attachments/assets/f6250153-1c6a-4336-b385-6ed03880bcd5)

![image](https://github.com/user-attachments/assets/e84519cf-6fbc-4275-bf28-c25512c384b5)

![image](https://github.com/user-attachments/assets/4b4f7202-73e3-47b6-b7d3-1a67b7eeb130)

**Comment by denvercal1234GitHub** [comment]
This should provide a temp fix: `remotes::install_version("ggplot2", "3.4.4")`

**Comment by samuel-marsh** [comment]
Hi @XIANZHE-LI 

If you update to the in dev version of scCustomize that will resolve the error.

```
devtools::install_github(repo = "samuel-marsh/scCustomize", ref = "release/2.2.0")
```

Best,
Sam
